### PR TITLE
Misc doc updates and refreshes

### PR DESCRIPTION
**Pull request by edvald**

### CODE REVIEW

**Comment by edvald** [comment]
Note: I dropped the FAQ in preference of the updated README, plus a bunch of it felt dated and irrelevant today.

File: docs/basics/stack-graph.md [hunk 1]
```diff
@@ -15,13 +21,16 @@ The Stack Graph is essentially an opinionated graph structure, with a handful of
...
 * **Test**: A test is also something you run and wait for to finish, similar to tasks, but with slightly different semantics and separate commands for execution. It can depend on services and tasks (but notably services and tasks cannot depend on tests).
 
-Each part of your stack _describes itself_ using a simple configuration file. Garden collects all those declarations, validates, and compiles them into a DAG (a directed acyclic graph, meaning it should have no circular dependencies).
+Each part of your stack _describes itself_ using a simple configuration file. Garden collects all those declarations, validates, and compiles them into a DAG (a _directed acyclic graph_, meaning it must have no circular dependencies).
 
 Importantly, what happens within each of the actions that the graph describes—building, deploying, running etc.—is completely pluggable via the providers. The Stack Graph is only opinionated in terms of flows and dependencies—_what_ should happen _when_—but the _how_ is pluggable.
 
+All the Garden plugins are currently built-in, but we will soon release a plugin SDK to allow any user to easily make their
```
Comment (ellenkorbes) [inline docs/basics/stack-graph.md]: Suggestion:
`built-in, but we`
↓
`built-in; we`

File: docs/using-garden/configuration-files.md [hunk 2]
```diff
@@ -275,56 +277,94 @@ Note that tasks that are not depended on by any services are not run by `garden
...
+type: container
+include:
+  - Dockerfile
+  - my-sources/**/*.py
+...
+```
+
+Here we only include the `Dockerfile` and all the `.py` files under `my-sources/`.
+
+`.gardenignore` do the opposite, which is to list all files/paths that you want to ignore. They work exactly
```
Comment (ellenkorbes) [inline docs/using-garden/configuration-files.md]: Missing word 'files':

`.gardenignore` files do the opposite

**Comment by edvald** [comment]
👍 thanks!

**Comment by edvald** [comment]
@ellenkorbes: Addressed your comments. Can you check and approve?

**Comment by ellenkorbes** [comment]
LGTM

**Pull request merged**